### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by TheKhanj**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/terminus/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: lib/health-indicator/http/http.health.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@ export class HttpHealthIndicator extends HealthIndicator {
   }
 
   /**
-   * Checks if the given url response in the given timeout
+   * Checks if the given url responses in the given timeout
```
Comment (BrunnerLivio) [inline lib/health-indicator/http/http.health.ts:94]: This is not a typo

**Review by BrunnerLivio — CHANGES_REQUESTED** [review]

File: lib/utils/checkPackage.util.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ import { Logger } from '@nestjs/common/services/logger.service';
 const MISSING_REQUIRED_DEPENDENCY = (names: string[], reason: string): string =>
   `The "${names.join('", "')}" package${
     names.length > 1 ? 's are' : ' is'
-  } missing. Please, make sure to install the librar${
+  } missing. Please, make sure to install the library${
```
[the remark this answers is not in the document]
Comment (TheKhanj) [reply lib/utils/checkPackage.util.ts]: Oh sorry for that :)

File: lib/health-indicator/http/http.health.ts (hunk 1 above)
Comment (TheKhanj) [reply lib/health-indicator/http/http.health.ts]: I'm not native, you know better 😁

**Comment by BrunnerLivio** [comment]
Thanks appreciated!

**Pull request merged**